wandb/sdk/internal/internal_api.py

```diff
@@ -2216,6 +2216,92 @@ def upsert_run(
             server_messages,
         )
 
+    @normalize_exceptions
+    def rewind_run(
+        self,
+        run_name: str,
+        metric_name: str,
+        metric_value: float,
+        program_path: Optional[str] = None,
+        entity: Optional[str] = None,
+        project: Optional[str] = None,
+        num_retries: Optional[int] = None,
+    ) -> dict:
+        """Rewinds a run to a previous state.
+
+        Arguments:
+            run_name (str): The name of the run to rewind
+            metric_name (str): The name of the metric to rewind to
+            metric_value (float): The value of the metric to rewind to
+            program_path (str, optional): Path to the program.
+            entity (str, optional): The entity to scope this project to.
+            project (str, optional): The name of the project
+            num_retries (int, optional): Number of retries
+        """
```

Comment:
The docstring should document its return value as well. It should either enumerate what one expects to be in the dictionary or write down how one can find out (by referencing some other documentation, maybe?)

Reply:
done - but this is not done in upsert_run currently

Reply:
thank you!

> this is not done in upsert_run currently

it's not a status quo that we should strive to keep :)